src/SCS.jl

```diff
@@ -16,7 +16,12 @@ function __init__()
     if is_unix()
         dlopen(Base.liblapack_name, RTLD_LAZY|RTLD_DEEPBIND|RTLD_GLOBAL)
     end
-    if !(vnum.major == 1 && vnum.minor == 1)
+    if is_linux()
+        if !(vnum.major == 1 && vnum.minor == 2)
```

Comment:
If the Julia code is compatible, there's no need to special case ``is_linux`` here. There should be one condition that tests major version == 1 and minor version == 1 or 2.